spec/models/docket_spec.rb

```diff
@@ -77,6 +77,38 @@
         end
       end
 
+      # This should probably actually test distribute_appeals
+      context "when there is a judge and the distribution is not_genpop" do
+        let(:other_judge) { create(:user, :judge, :with_judge_team) }
+
+        let(:cavc_remand) { cavc_appeal.cavc_remand }
+        let(:source_appeal) { cavc_remand.source_appeal }
+        let(:judge_decision_review_task) do
+          source_appeal.tasks.where(
+            type: "JudgeDecisionReviewTask",
+            status: "completed"
+          ).first
+        end
+
+        context "when called for another judge who has no affinity" do
+          subject { DirectReviewDocket.new.appeals(ready: true, genpop: "not_genpop", judge: other_judge) }
+
+          it "returns no appeals" do
+            expect(subject.count.size).to eq 0 # still don't get why this is needed
```

Comment:
```suggestion
            expect(subject.count.size).to eq 0
```

The comments in the existing `count` method here address this. tl;dr, the weird behavior where `.count` returns a hash is because there's a `group_by` in the query.

```ruby
  def count(priority: nil, ready: nil)
    # The underlying scopes here all use `group_by` statements, so calling
    # `count` on `appeals` will return a hash. To get the number of appeals, we
    # can pluck the ids and ask for the size of the resulting array.
    # See the docs for ActiveRecord::Calculations
    appeals(priority: priority, ready: ready).ids.size
  end
```